dist/api_data.py

```diff
@@ -1057,18 +1040,45 @@ def __ge__(self, other):
     ]),
 ]
 
-wiredtiger_open_statistics_log_configuration = [
-    Config('statistics_log', '', r'''
-        log any statistics the database is configured to maintain, to a file. See @ref
-        statistics for more information. Enabling the statistics log server uses a session from
-        the configured session_max''',
+chunk_cache_configuration_common = [
+    Config('pinned', '', r'''
+        List of "table:" URIs exempt from cache eviction. Capacity config overrides this,
+        tables exceeding capacity will not be fully retained. Table names can appear
+        in both this and the preload list, but not in both this and the exclude list.
+        Duplicate names are allowed.''',
+        type='list'),
+]
+connection_reconfigure_chunk_cache_configuration = [
+    Config('chunk_cache', '', r'''
+        chunk cache reconfiguration options''',
+        type='category', subconfig=chunk_cache_configuration_common)
+]
+wiredtiger_open_chunk_cache_configuration = [
+    Config('chunk_cache', '', r'''
+        chunk cache configuration options''',
         type='category', subconfig=
-        statistics_log_configuration_common + [
-        Config('path', '"."', r'''
-            the name of a directory into which statistics files are written. The directory
-            must already exist. If the value is not an absolute path, the path is relative to
-            the database home (see @ref absolute_path for more information)''')
-        ])
+        chunk_cache_configuration_common + [
+        Config('capacity', '10GB', r'''
+            maximum memory or storage to use for the chunk cache''',
+            min='0', max='100TB'),
+        Config('chunk_cache_evict_trigger', '90', r'''
+            chunk cache percent full that triggers eviction''',
+            min='0', max='100'),
+        Config('chunk_size', '1MB', r'''
+            size of cached chunks''',
+            min='512KB', max='100GB'),
+        Config('device_path', '', r'''
+            the absolute path to the file system or a block device used as cache location'''),
+        Config('enabled', 'false', r'''
+            enable chunk cache''',
+            type='boolean'),
+        Config('hashsize', '1024', r'''
+            number of buckets in the hashtable that keeps track of objects''',
+            min='64', max='1048576'),
+        Config('type', 'FILE', r'''
+            cache location, defaults to the file system.''',
+            choices=['FILE', 'DRAM']),
```

Comment:
We still want `undoc=True` here.

Reply:
Nice catch. I may have accidentally removed it, trying to test something.

Reply:
Updated in [d9037e5](https://github.com/wiredtiger/wiredtiger/pull/9466/commits/d9037e504ffd73918da9c25373f78b62a0746b77)